project.clj

```diff
@@ -1,10 +1,10 @@
 ;; Please don't bump the library version by hand - use ci.release-workflow instead.
-(defproject com.nedap.staffing-solutions/components.pedestal "2.0.0"
+(defproject com.nedap.staffing-solutions/components.pedestal "2.0.1-alpha3"
   ;; Please keep the dependencies sorted a-z.
   :dependencies [[com.grzm/component.pedestal "0.1.7"
                   :exclusions [ring/ring-codec]]
-                 [com.nedap.staffing-solutions/utils.modular "2.1.0"]
-                 [com.nedap.staffing-solutions/speced.def "2.0.0"
+                 [com.nedap.staffing-solutions/utils.modular "2.2.0"]
+                 [com.nedap.staffing-solutions/speced.def "2.1.1"
                   :exclusions [org.clojure/spec.alpha]]
```

Comment:
We generally replace the :exclusions with managed-dependencies too (so we're explicit about what we expect, instead of relying on transient dep definitions).

Could you update this as well?